### PR TITLE
Add Axelar chain

### PR DESCRIPTION
**Pull request by CharlesJUDITH**

### CODE REVIEW

File: axelar/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Run on Akash with the [example deploy.yml](./deploy.yml)
+- Run locally using the [example docker-compose.yml](./docker-compose.yml)
+
+## Chain information
+
+The [Cosmos Chain Registry](https://github.com/cosmos/chain-registry) publishes up to date chain info for Axelar.
+
+|Variable|Value|
+|---|---|
+|`CHAIN_JSON`|`https://raw.githubusercontent.com/axelarnetwork/axelarate-community/main/resources/mainnet/genesis.json`|
```
Comment (tombeynon) [inline axelar/README.md:23]:
```suggestion
|`CHAIN_JSON`|`https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/chain.json`|
```

**Review by tombeynon — CHANGES_REQUESTED** [review]

**Comment by tombeynon** [comment]
`ghcr.io/akash-network/cosmos-omnibus:v0.4.4-rc2-axelar-v0.34.0`

File: axelar/build.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        REPOSITORY: https://github.com/akash-network/node
+        BUILD_IMAGE: binary
+        BINARY_URL: https://github.com/axelarnetwork/axelar-core/releases/download/v0.34.0/axelard-linux-amd64-v0.34.0
+    ports:
+      - '26656:26656'
+      - '26657:26657'
+      - '1317:1317'
+    environment:
+      - MONIKER=node_1
+      - CHAIN_JSON=https://raw.githubusercontent.com/axelarnetwork/axelarate-community/main/resources/mainnet/genesis.json
```
[the remark this answers is not in the document]
Comment (CharlesJUDITH) [reply axelar/build.yml]: added in https://github.com/akash-network/cosmos-omnibus/commit/8ec6aec59a2ed21fdf1ccb18670d24ed23a9dbed

File: axelar/build.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+version: '3.4'
+
+services:
+  node_1:
+    build:
+      context: ../
+      args:
+        PROJECT: axelar
+        VERSION: v0.34.0
+        REPOSITORY: https://github.com/akash-network/node
```
[the remark this answers is not in the document]
Comment (CharlesJUDITH) [reply axelar/build.yml]: added in https://github.com/akash-network/cosmos-omnibus/commit/8ec6aec59a2ed21fdf1ccb18670d24ed23a9dbed

File: axelar/README.md (hunk 1 above)
Comment (CharlesJUDITH) [reply axelar/README.md]: added in https://github.com/akash-network/cosmos-omnibus/commit/8ec6aec59a2ed21fdf1ccb18670d24ed23a9dbed

File: axelar/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Akash
```
[the remark this answers is not in the document]
Comment (CharlesJUDITH) [reply axelar/README.md]: added in https://github.com/akash-network/cosmos-omnibus/commit/8ec6aec59a2ed21fdf1ccb18670d24ed23a9dbed

File: axelar/deploy.yml [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+version: "2.0"
+
+services:
+  node:
+    image: ghcr.io/akash-network/cosmos-omnibus:v0.4.3-axelar-v0.34.0
+    env:
+      - MONIKER=my-moniker-1
+      - CHAIN_JSON=https://raw.githubusercontent.com/axelarnetwork/axelarate-community/main/resources/mainnet/genesis.json
```
[the remark this answers is not in the document]
Comment (CharlesJUDITH) [reply axelar/deploy.yml]: added in https://github.com/akash-network/cosmos-omnibus/commit/8ec6aec59a2ed21fdf1ccb18670d24ed23a9dbed

File: axelar/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+services:
+  node_1:
+    image: ghcr.io/akash-network/cosmos-omnibus:v0.4.3-axelar-v0.34.0
+    ports:
+      - '26656:26656'
+      - '26657:26657'
+      - '1317:1317'
+    environment:
+      - MONIKER=node_1
+      - CHAIN_JSON=https://raw.githubusercontent.com/axelarnetwork/axelarate-community/main/resources/mainnet/genesis.json
```
[the remark this answers is not in the document]
Comment (CharlesJUDITH) [reply axelar/docker-compose.yml]: Added in https://github.com/akash-network/cosmos-omnibus/commit/8ec6aec59a2ed21fdf1ccb18670d24ed23a9dbed

**Review by tombeynon — APPROVED** [review]

**Comment by CharlesJUDITH** [comment]
> @CharlesJUDITH can you avoid force pushing if possible 🙏🏻

sure

**Pull request merged**